bin/fin

```diff
@@ -208,24 +208,29 @@ PATH="$CONFIG_BIN_DIR:$PATH"
 DOCKER_NATIVE="${DOCKER_NATIVE:-0}"
 
 #---------------------------- URL references --------------------------------
-URL_REPO="https://raw.githubusercontent.com/docksal/docksal"
-URL_REPO_UI="https://github.com/docksal/docksal"
-URL_REPO_DRUPAL7="https://github.com/docksal/drupal7.git"
-URL_REPO_DRUPAL8="https://github.com/docksal/drupal8.git"
-URL_REPO_DRUPAL8COMPOSER="https://github.com/docksal/boilerplate-drupal8-composer.git"
-URL_REPO_WORDPRESS="https://github.com/docksal/wordpress.git"
-URL_REPO_MAGENTO="https://github.com/docksal/magento.git"
-URL_REPO_GRAV="https://github.com/docksal/example-grav.git"
-URL_REPO_GATSBY="https://github.com/docksal/example-gatsby.git"
-URL_REPO_LARAVEL="https://github.com/docksal/example-laravel.git"
-URL_REPO_HUGO="https://github.com/docksal/example-hugo.git"
-URL_REPO_SYMFONY_SKELETON="https://github.com/docksal/example-symfony-skeleton.git"
-URL_REPO_SYMFONY_WEBAPP="https://github.com/docksal/example-symfony-webapp.git"
-URL_REPO_BACKDROP="https://github.com/docksal/example-backdrop.git"
-URL_ADDONS_HOSTING="https://raw.githubusercontent.com"
-URL_ADDONS_REPO="$URL_ADDONS_HOSTING/docksal/addons"
+REPO_OWNER="docksal"
```

Comment:
What is the purpose of this constant? It's not like we are going to change it abruptly :)

Reply:
Can we please revert these the changes unrelated to Docksal RC Version for now?